### ISSUE TITLE
Make it easier to configure HiRA options in Point-Stat.

### ISSUE THREAD

**Issue by JohnHalleyGotway** [dtcenter/MET#1400]
## Describe the Enhancement ##
This enhancement request arose from a met-help question from John Raby:
https://rt.rap.ucar.edu/rt/Ticket/Display.html?id=95790

He's trying out the HiRA methodology in Point-Stat and found the prob_cat_thresh settings confusing. He configured Point-Stat to verify multiple variables, including TMP, RH, and winds. He defined cat_thresh for each and requested categorical counts and statistics.

The prob_cat_thresh setting is used in the computation of the Ranked Probability Score (RPS) line type. He assumed those same cat_thresh thresholds would be applied when computing RPS for HiRA, but they are not. Instead, the thresholds are defined by hira.prob_cat_thresh. If hira.prob_cat_thresh is left empty AND climo bins are provided, then Point-Stat will use those thresholds instead. But if none of those are the case, then point-stat will just skip the computation of RPS.

If HiRA RPS output is desired when not using climatologies, then setting the prob_cat_thresh thresholds well becomes very cumbersome. The user would need to define prob_cat_thresh thresholds separately for each field... meaning they'd need to re-define the HiRA dictionary settings for each variable! And that's very cumbersome.

To make this easier, tweak the logic as follows. When computing the HiRA RPS scores, if prob_cat_thresh is empty AND there's no climo bins, simply use the observation cat_thresh thresholds to define RPS. This change will enable users to define the hira dictionary only once and apply it to multiple variables.

### Time Estimate ###
4 hours

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
- No sub-issues required.

### Relevant Deadlines ###
None

### Funding Source ###
None

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [x] Select **scientist(s)** or **no scientist** required: Did not assign a scientist but did ask for Tara to do the pull request review.

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [X] Review **projects** and select relevant **Repository** and **Organization** ones
- [X] Select **milestone**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [X] [METplus](https://github.com/NCAR/METplus/issues/new/choose), [MET](https://github.com/NCAR/MET/issues/new/choose), [METdb](https://github.com/NCAR/METdb/issues/new/choose), [METviewer](https://github.com/NCAR/METviewer/issues/new/choose), [METexpress](https://github.com/NCAR/METexpress/issues/new/choose), [METcalcpy](https://github.com/NCAR/METcalcpy/issues/new/choose), [METplotpy](https://github.com/NCAR/METplotpy/issues/new/choose)
No impacts.

## Enhancement Checklist ##
See the [METplus Workflow](https://ncar.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [X] Complete the issue definition above.
- [X] Fork this repository or create a branch of **develop**.
Branch name:  `feature_<Issue Number>_<Description>`
- [X] Complete the development and test your changes.
- [X] Add/update unit tests.
- [X] Add/update documentation.
- [X] Push local changes to GitHub.
- [X] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [x] Iterate until the reviewer(s) accept and merge your changes.
- [x] Delete your fork or branch.
- [x] Close this issue.
